eval/asserts/fix-plan-impact-prompt.test.mjs

```
@@ -0,0 +1,25 @@
+import assert from 'node:assert/strict';
+import { readFileSync } from 'node:fs';
+import test from 'node:test';
+
+const promptPaths = [
+  new URL('../prompts/fix-plan-impact-closure-primary.phase1.md', import.meta.url),
```

ESLint

[error] 'URL' is not defined.

(no-undef)